person.rb

```diff
@@ -18,4 +20,8 @@ def of_age?
   def can_use_services?
     of_age? || @parent_permission
   end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] This should not be a private method. Kindly make it public, so the decorator can be tested.